docs/getting_started/components/hls.md

```diff
@@ -37,6 +37,10 @@ After a meeting that integrates an HLS component (with the persistent option set
 To manage this recording, use the [Recording API](../../for_developers/api_reference/rest_api#tag/recording).
 The recording is also available as HLS Video On Demand (VOD) [API](../../for_developers/api_reference/rest_api#tag/recording/operation/getRecordingContent).
 
+## Store HLS stream on S3
```

Comment:
We also need to add s3 configuration options in the section above

Reply:
Currently, we don't support s3 streams in recordings so I don't see the point.